src/lifetracker/logic/LogicImpl.java

```diff
@@ -42,7 +46,23 @@ public ExecuteResult executeCommand(String commandString) {
         if (commandString.equals("exit")) {
             return runResult;
 
-        } else {
+        }
+        
+        else if (commandContent[0].equals("saveat")) {
```

Comment:
Can we have a seperate enum for saveat? The UI will end up showing an empty list if we simply leave it as "DISPLAY"